extensions/api/util.py

```diff
@@ -75,6 +75,8 @@ def build_parameters(body, chat=False):
         'silero_tts_voice_pitch': body.get('silero_tts_voice_pitch', 'medium'),
         'silero_tts_voice_speed': body.get('silero_tts_voice_speed', 'medium'),
 
+        '11labs_tts_voice' : body.get('11labs_tts_voice', 'Alice'),
```

Comment:
we should use the fullname and no numbers. `elevenlabs_tts_voice`

Also, we should have a `elevenlabs_tts_enable` or similar, so that we can have one of 3 options: no tts, silero only, elevenlabs then silero.

BUT!!! Because we are configuring elevenlabs inside of silero, then we should use the names `silero_elevenlabs_enabled`, for example.